# Define the goal
geometry_msgs/PoseStamped place_pose
string place_frameID
---
# Define the result
bool succeed
---
# Define a feedback message
float32 percent_complete